app/views/companies/show.html.haml

```diff
@@ -1,51 +1,49 @@
-%header.entry-header
-  %h1.entry-title= @company.name
-  .post-title-divider
+%div{ :id => "#{@company.class.name.downcase}-#{@company.id}", class: "show-item #{@company.class.name.downcase} #{@company.class.name.downcase}-#{@company.id}" }
```

Comment:
because I'm thinking too far ahead. :-)  

I'm thinking of the general pattern (`"#{class-id}"`) that I think we should implement for all views so that we have the CSS 'hooks' available for all views.   This is a general pattern I've used in web apps (not just Ruby), so I'm in the habit of thinking/typing that way.  

That said, I'm fine making this `"company-#{@company.id}"` for now if you'd like. But I'm betting that we'll need to come back and refactor it later when we need more consistent CSS styling applied to views.